packages/amplify-provider-awscloudformation/src/__tests__/iterative-deployment/stack-progress-printer.test.ts

```diff
@@ -2,7 +2,16 @@ import { StackProgressPrinter } from '../../iterative-deployment/stack-progress-
 import { IStackProgressPrinter } from '../../iterative-deployment/stack-event-monitor';
 import columnify from 'columnify';
 
-const printer: IStackProgressPrinter = new StackProgressPrinter();
+const eventMap = {
+  projectName: 'test',
+  envName: 'dev',
+  rootStackName: 'root-app',
+  rootResources: [{ key: 'test-app', category: 'api' }],
+  categories: [{ name: 'api', size: 2 }],
+  eventToCategories: new Map(),
+};
+
+const printer: IStackProgressPrinter = new StackProgressPrinter(eventMap);
 jest.mock('columnify');
 
```

Comment:
**columnify** is not being called when testing locally due to TTY detection;
causing this test to fail:
![image](https://user-images.githubusercontent.com/110861985/192401947-0f202bc8-6c28-4617-8168-64c5abcc6d82.png)

Reply:
Related to this , stack-progres-printer.ts

![image](https://user-images.githubusercontent.com/110861985/192401991-d85bb33c-f886-4fc9-865d-d32c7188cee7.png)

Reply:
After a bit more review; this test is only for the "defaultLogs".
A new set of tests need to be designed & added for the pretty printer.